docs/kb/semgrep-ci/scan-monorepo-in-parts.md

```diff
@@ -19,22 +19,28 @@ As such, it can be helpful to scan a monorepo in parts for multiple reasons:
 
 When scanning a repo with Semgrep in CI, the base command is `semgrep ci`. To understand this default setup for your source code manager (SCM) and CI provider, see [Getting started with Semgrep in continuous integration (CI)](/deployment/add-semgrep-to-ci).
 
-There are two features provided by Semgrep to split up a repo. Consider a monorepo named `monorepo` with four main modules
+There are two features provided by Semgrep to split up a repo. Consider a monorepo named `monorepo` with four main modules:
 
-    /src/moduleA
-    /src/moduleB
-    /src/moduleC
-    /src/moduleD
+ ```
```

Comment:
@khorne3 not sure why it would make sense to remove this?

Reply:
@armchairlinguist Sorry, I was trying to indicate that there's an extra space in front of the code block formatter. I'll just fix, because it seems I can't suggest deleting just the space.